Directory.Build.props

```diff
@@ -1,5 +1,6 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <CL_MPCount>$(NumberOfCores)</CL_MPCount>
+    <VCPkgEnabled>false</VCPkgEnabled>
```

Comment:
Microsoft VCPkg tool generates a lot of warnings when building for Checked configurations and in case of Release and Debug configurations up to 10 vcpkg servers may be running as children of VS taking up to 1 GB of memory. This setting disables vcpkg tool in coreclr builds both from command line and from VS.

Reply:
We've unfortunately overloaded the term Vcpkg here -- the "vcpkg servers" you're talking about are a completely different thing than the Vcpkg tool and those won't be affected by this setting.

Reply:
@ras0219-msft Thanks for clarification!